css/styles.css

```diff
@@ -57,38 +57,57 @@ body {
     padding: 0 20px;
 }
 
+.main-content {
+    width: 485px;
+    margin: 0 auto;
+    padding-top: 50px;
+}
+
+.footer {
+    background-color: #eeeeee;
+    color: rgba(206, 206, 206, 0.54);
+    position: fixed;
+    left: 0;
+    bottom: 0;
+    width: 100%;
+}
+
 /* Header Navigation
 =================================================*/
-header nav {
+.header-nav {
     text-align: right;
     padding-top: 20px;
 }
 
-header nav ul {
+.header .header-nav ul {
```

Comment:
You shouldn't need to nest these classes.  `.header-nav ul {}` should be specific enough